zerver/views/registration.py

```diff
@@ -539,38 +539,33 @@ def find_account(request: HttpRequest) -> HttpResponse:
     url = reverse('zerver.views.registration.find_account')
 
     emails = []  # type: List[str]
+    emails_sent = []  # type: List[str]
+    emails_ignore = []  # tyep: List[str]
     if request.method == 'POST':
         form = FindMyTeamForm(request.POST)
         if form.is_valid():
             emails = form.cleaned_data['emails']
-            for user in UserProfile.objects.filter(
-                    delivery_email__in=emails, is_active=True, is_bot=False,
-                    realm__deactivated=False):
-                context = common_context(user)
-                context.update({
-                    'email': user.delivery_email,
-                })
-                send_email('zerver/emails/find_team', to_user_ids=[user.id], context=context,
+            for user in UserProfile.objects.filter(delivery_email__in=emails):
+                if user.is_active is True and user.is_bot is False and user.realm.deactivated is False:
+                    context = common_context(user)
+                    context.update({
+                        'email': user.delivery_email,
+                    })
+                    send_email('zerver/emails/find_team', to_user_ids=[user.id], context=context,
+                               from_address=FromAddress.SUPPORT)
+                    if user.delivery_email not in emails_sent:
+                        emails_sent.append(user.delivery_email)
+                else:
+                    emails_ignore.append(user.delivery_email)
+            for email in list((set(emails).difference(set(emails_sent))).difference(set(emails_ignore))):
+                send_email('zerver/emails/find_team', to_emails=[email], context=None,
                            from_address=FromAddress.SUPPORT)
 
             # Note: Show all the emails in the result otherwise this
             # feature can be used to ascertain which email addresses
             # are associated with Zulip.
-            data = urllib.parse.urlencode({'emails': ','.join(emails)})
-            return redirect(url + "?" + data)
     else:
         form = FindMyTeamForm()
-        result = request.GET.get('emails')
-        # The below validation is perhaps unnecessary, in that we
-        # shouldn't get able to get here with an invalid email unless
-        # the user hand-edits the URLs.
-        if result:
-            for email in result.split(','):
-                try:
-                    validators.validate_email(email)
-                    emails.append(email)
-                except ValidationError:
-                    pass
```

Comment:
What's your reasoning behind deleting this?

Reply:
So that part belongs to the `GET` method for the page which was used to fetch list of emails from the URL as parameters. This would allow some user to manually edit the contents of the URL, the following piece was just to make sure that the emails in the URL are valid or not. Since we have don't want to pass the list of emails as parameters to the follow up page it won't be required.